frontend/src/model/beta/releases/EditableRelease.tsx

```diff
@@ -35,8 +35,8 @@ export default function EditableRelease({ release }: EditableReleaseProps) {
     setSemver(release.semver)
     setReleaseNotes(release.notes)
     setIsMinorRelease(!!release.minor)
-    setArtefacts([]) // TODO - Reset the release artefact files (BAI-1026)
-    setArtefactsMetadata([])
+    setFiles([]) // TODO - Reset the release file files (BAI-1026)
```

Comment:
This comment no longer reads correctly. Change to `// TODO - Reset the release files (BAI-1026)`?